aws/modules/infrastructure_modules/vpc/network_acls.tf

```diff
@@ -0,0 +1,201 @@
+  cidr_block      = lookup(var.private_inbound_acl_rules[count.index], "cidr_block", null)
+  ipv6_cidr_block = lookup(var.private_inbound_acl_rules[count.index], "ipv6_cidr_block", null)
+}
+
+resource "aws_network_acl_rule" "database_outbound" {
+  count = var.create_database_dedicated_network_acl ? length(var.database_outbound_acl_rules) : 0
+
+  network_acl_id = aws_network_acl.database[0].id
+
+  egress          = true
+  rule_number     = var.database_outbound_acl_rules[count.index]["rule_number"]
+  rule_action     = var.database_outbound_acl_rules[count.index]["rule_action"]
+  protocol        = var.database_outbound_acl_rules[count.index]["protocol"]
+  from_port       = lookup(var.database_outbound_acl_rules[count.index], "from_port", null)
+  to_port         = lookup(var.database_outbound_acl_rules[count.index], "to_port", null)
+  icmp_code       = lookup(var.database_outbound_acl_rules[count.index], "icmp_code", null)
+  icmp_type       = lookup(var.database_outbound_acl_rules[count.index], "icmp_type", null)
+  cidr_block      = lookup(var.database_outbound_acl_rules[count.index], "cidr_block", null)
+  ipv6_cidr_block = lookup(var.database_outbound_acl_rules[count.index], "ipv6_cidr_block", null)
+}
+
+
+################################################################################
+# Network Firewall Subnet Network ACLs
+################################################################################
+
+resource "aws_network_acl" "network_firewall" {
+  count = var.create_network_firewall_dedicated_network_acl ? 1 : 0
+
+  vpc_id     = module.vpc.vpc_id
+  subnet_ids = aws_subnet.network_firewall[*].id
+
+  tags = merge(
+    { "Name" = "${var.vpc_name}-network-firewall-acl" },
+    var.tags
+  )
+}
+
+resource "aws_network_acl_rule" "network_firewall_inbound" {
+  count = var.create_network_firewall_dedicated_network_acl ? length(var.network_firewall_inbound_acl_rules) : 0
+
+  network_acl_id = aws_network_acl.network_firewall[0].id
+
+  egress          = false
+  rule_number     = var.network_firewall_inbound_acl_rules[count.index]["rule_number"]
+  rule_action     = var.network_firewall_inbound_acl_rules[count.index]["rule_action"]
+  protocol        = var.network_firewall_inbound_acl_rules[count.index]["protocol"]
+  from_port       = lookup(var.network_firewall_inbound_acl_rules[count.index], "from_port", null)
+  to_port         = lookup(var.network_firewall_inbound_acl_rules[count.index], "to_port", null)
+  icmp_code       = lookup(var.network_firewall_inbound_acl_rules[count.index], "icmp_code", null)
+  icmp_type       = lookup(var.network_firewall_inbound_acl_rules[count.index], "icmp_type", null)
+  cidr_block      = lookup(var.network_firewall_inbound_acl_rules[count.index], "cidr_block", null)
+  ipv6_cidr_block = lookup(var.network_firewall_inbound_acl_rules[count.index], "ipv6_cidr_block", null)
+}
+
+resource "aws_network_acl_rule" "firewall_outbound" {
```

Comment:
The rest are prefixed with `network_` can we add that to this too please?

Reply:
done